tools/export/exporters.py

```diff
@@ -130,9 +130,13 @@ def make_key(self, src):
         Positional Arguments:
         src - the src's location
         """
-        key = basename(dirname(src))
-        if key == ".":
-            key = basename(realpath(self.export_dir))
+        rel_path = relpath(src, self.resources.file_basepath[src])
+        path_list = os.path.normpath(rel_path).split(os.sep)
+        assert path_list >= 1
```

Comment:
I created #3574 to fix this. This assert triggers in the case where the list is empty, as this is invalid and would cause an exception later anyway. This assert doesn't really matter though and I can remove it if you would like.